src/VisualStudio/Core/Def/Interactive/VsInteractiveWindowPackage.cs

```diff
@@ -71,8 +71,8 @@ private static void SetErrorHandlers(Assembly assembly)
             var handlerSetter = type.GetDeclaredMethod("set_Handler");
             var nonFatalHandlerSetter = type.GetDeclaredMethod("set_NonFatalHandler");
 
-            handlerSetter.Invoke(null, new object[] { new Action<Exception>(FailFast.OnFatalException) });
-            nonFatalHandlerSetter.Invoke(null, new object[] { new Action<Exception>(WatsonReporter.ReportNonFatal) });
+            handlerSetter.Invoke(null, new object[] { new Action<Exception, string>(FailFast.OnFatalException) });
+            nonFatalHandlerSetter.Invoke(null, new object[] { new Action<Exception, string>(WatsonReporter.ReportNonFatal) });
```

Comment:
@tmat wasn't certain about this since reflection is involved.  do you know what is needed for interactive window?